<launch>
  <arg name="camera_name" default="thermal"/>

  <node pkg="nodelet" type="nodelet" name="flir_nodelet_manager"  args="manager" 
    output="screen"/>

  <node pkg="nodelet" type="nodelet" name="flir_nodelet" 
    args="load flir_ros/FlirRos flir_nodelet_manager" output="screen">
        <param name="device_name" value="/dev/mmpug/flir_boson_video"/>
        <param name="raw" value="true" type="bool" />
        <param name="send_every_n" value="1" type="int" />

        <param name="width" value="640" type="int"/>
        <param name="height" value="512" type="int"/>

        <param name="camera_name" value="$(arg camera_name)"/>
        <param name="intrinsic_url" value="package://flir_ros/data/camera_info/example/thermal_caminfo_flir_320.yaml"/>
        <remap from="/set_camera_info" to="$(arg camera_name)/set_camera_info"/>
  </node>
</launch>
